web/src/people.js

```diff
@@ -1328,8 +1328,6 @@ export function extract_people_from_message(message) {
             email: person.email,
             user_id,
             full_name: person.full_name,
-            is_admin: person.is_realm_admin || false,
-            is_bot: person.is_bot || false,
```

Comment:
I think this should instead be hardcoding these to false. I'm not sure, but I kinda suspect this commit alone breaks the app and it's only fixed when you introduce `make_user` here later; it might make sense to just squash to avoid the intermediate state -- the actual problem here is that this is not a full user object.